### PR TITLE
Add hints to engineering onboarding

### PR DESCRIPTION
**Pull request by daxmc99**
Some slight hints to make the engineering onboarding a bit more clear and provide some guidance on using Postgres.

### CODE REVIEW

File: handbook/engineering/onboarding.md [hunk 1]
```diff
@@ -31,6 +31,9 @@ As you are completing these tasks, you may notice documentation, processes, or c
 - Read through the rest of the engineering handbook to learn more about how we operate.
 - Setup your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/local_development.md#step-1-install-dependencies). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
 - Make yourself an admin on sourcegraph.com and sgdev.org by updating the database directly (this is not what a normal user would do, but doing it this way will expose you to useful knowledge). Relevant documentation:
+  - Hint: Make sure you have an account at these locations before editing the database
```
Comment (nicksnyder) [inline handbook/engineering/onboarding.md:34]: What locations is this referring to? Is there a missing link here?

Comment (daxmc99) [reply handbook/engineering/onboarding.md:34]: `sourcegraph.com`, `sourcegraph.sgdev.org`, & `k8s.sgdev.org`
They are referenced on the lines below that point to deployments. 
https://about.sourcegraph.com/handbook/engineering/deployments
I think both these locations would benefit from being changed to links.

**Comment by slimsag** [comment]
test

**Comment by slimsag** [comment]
hello

File: handbook/engineering/onboarding.md [hunk 2]
```diff
@@ -31,6 +31,9 @@ As you are completing these tasks, you may notice documentation, processes, or c
 - Read through the rest of the engineering handbook to learn more about how we operate.
 - Setup your [local development environment](https://github.com/sourcegraph/sourcegraph/blob/master/doc/dev/local_development.md#step-1-install-dependencies). If you encounter any issues, ask for help in Slack and then update the documentation to reflect the resolution (so the next engineer that we hire doesn't run into the same problem).
 - Make yourself an admin on sourcegraph.com and sgdev.org by updating the database directly (this is not what a normal user would do, but doing it this way will expose you to useful knowledge). Relevant documentation:
+  - Make sure you have an account on our deployments before editing the database [sourcegraph.com](https://sourcegraph.com), [sourcegraph.sgdev.org](https://sourcegraph.sgdev.org), [k8s.sgdev.org](https://k8s.sgdev.org)
```
Comment (daxmc99) [inline handbook/engineering/onboarding.md:34]: @nicksnyder PTAL

Comment (nicksnyder) [reply handbook/engineering/onboarding.md:34]:
```suggestion
  - Make sure you have an account on our deployments ([sourcegraph.com](https://sourcegraph.com), [sourcegraph.sgdev.org](https://sourcegraph.sgdev.org), [k8s.sgdev.org](https://k8s.sgdev.org)) before editing the database.
```

File: handbook/engineering/onboarding.md [hunk 3]
```diff
@@ -31,6 +31,9 @@ As you are completing these tasks, you may notice documentation, processes, or c
...
 - Make yourself an admin on sourcegraph.com and sgdev.org by updating the database directly (this is not what a normal user would do, but doing it this way will expose you to useful knowledge). Relevant documentation:
+  - Make sure you have an account on our deployments ([sourcegraph.com](https://sourcegraph.com), [sourcegraph.sgdev.org](https://sourcegraph.sgdev.org), [k8s.sgdev.org](https://k8s.sgdev.org)) before editing the database.
   - [Our deployments](deployments.md)
   - [How do I access the Sourcegraph database?](https://docs.sourcegraph.com/admin/faq#how-do-i-access-the-sourcegraph-database)
+  - PostgreSQL Hint: Consider using a [transaction](https://www.postgresql.org/docs/current/tutorial-transactions.html) so that you can ROLLBACK your actions in case of a mistake
 - Start working on the starter tasks that your manager has assigned you.
+`
```
Comment (unknwon) [inline handbook/engineering/onboarding.md:39]: Hmm, this is ... unexpected?

Comment (unknwon) [reply handbook/engineering/onboarding.md:39]: Anyway, PR sent: https://github.com/sourcegraph/about/pull/918

**Pull request merged**